spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfigurationTests.java

```diff
@@ -64,12 +69,33 @@ public void keepAliveValue_custom() {
 			GcpPubSubProperties props = ctx.getBean(GcpPubSubProperties.class);
 			assertThat(props.getKeepAliveIntervalMinutes()).isEqualTo(2);
 
-			TransportChannelProvider tcp = ctx.getBean(TransportChannelProvider.class);
-			assertThat(((InstantiatingGrpcChannelProvider) tcp).getKeepAliveTime().toMinutes())
+			TransportChannelProvider subscriberTcp = ctx.getBean("subscriberTransportChannelProvider", TransportChannelProvider.class);
+			TransportChannelProvider publisherTcp = ctx.getBean("publisherTransportChannelProvider", TransportChannelProvider.class);
+			assertThat(((InstantiatingGrpcChannelProvider) subscriberTcp).getKeepAliveTime().toMinutes())
+					.isEqualTo(2);
+			assertThat(((InstantiatingGrpcChannelProvider) publisherTcp).getKeepAliveTime().toMinutes())
 					.isEqualTo(2);
 		});
 	}
 
+	@Test
+	public void maxInboundMessageSize_default() {
+		ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+				.withConfiguration(AutoConfigurations.of(GcpPubSubAutoConfiguration.class))
+				.withUserConfiguration(TestConfig.class);
+
+		contextRunner.run(ctx -> {
+
+			TransportChannelProvider subscriberTcp = ctx.getBean("subscriberTransportChannelProvider", TransportChannelProvider.class);
+			assertThat(FieldUtils.readField(subscriberTcp, "maxInboundMessageSize", true))
+					.isEqualTo(20 << 20);
```

Comment:
define a 20MB constant maybe? Bit shifting is a bit obfuscated.

Reply:
It's a copy-paste from the client library.